From 64897e97edeefc7fbc89b07aaece760009ba76a0 Mon Sep 17 00:00:00 2001
From: "Chia-Wei, Wang" <chiawei_wang@aspeedtech.com>
Date: Mon, 21 Dec 2020 16:27:20 +0800
Subject: [PATCH] serial: 8250: Fix RX DMA time out property

Fix the typo that passing the TX DMA time out property
to the RX configuration.

Signed-off-by: Chia-Wei, Wang <chiawei_wang@aspeedtech.com>
---
 drivers/tty/serial/8250/8250_aspeed.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/tty/serial/8250/8250_aspeed.c b/drivers/tty/serial/8250/8250_aspeed.c
index 1dc798298fca..fc14c86d3761 100644
--- a/drivers/tty/serial/8250/8250_aspeed.c
+++ b/drivers/tty/serial/8250/8250_aspeed.c
@@ -266,7 +266,7 @@ static int ast8250_startup(struct uart_port *port)
 		}
 
 		rc = aspeed_udma_request_rx_chan(dma->ch, dma->rx_addr,
-				dma->rx_rb, dma->rx_rbsz, ast8250_dma_rx_complete, port, dma->tx_tmout_dis);
+				dma->rx_rb, dma->rx_rbsz, ast8250_dma_rx_complete, port, dma->rx_tmout_dis);
 		if (rc) {
 			dev_err(port->dev, "failed to request DMA RX channel\n");
 			goto free_dma_n_out;
-- 
2.17.1

